.changeset/little-apples-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/db': patch
+---
+
+Adds support for enum support for text columns in Astro DB tables.
```

Comment:
I would love to see a more comprehensive changeset. Considering it seems to be a new feature, we should at least have a snippet that shows how it works